### PR TITLE
More branding and fixes

### LINKED ISSUE #28: Confirmation Email Needs Branding - currently refers to KerbalStuff
BTW the text of the confirmation email still refers to the site as KerbalStuff. Not a system breaking bug but should be addressed sometime.

from the forum: http://forum.kerbalspaceprogram.com/index.php?/topic/132186-spacedock-dev-thread-the-kerbalstuff-replacement-site-now-live/&do=findComment&comment=2413736

### LINKED ISSUE #37: Branding during mod creation is messed up
Via Reddit:

> When you add a new mod it says "Thanks for using KerbalStuff" in the default about info.

### PR DESCRIPTION
**Pull request by Olympic1**
Revert back #40 to fix some problems and some more branding.

Fixes #28 Fixes #37 Fixes #40 Fixes #43 Fixes #44 Fixes #45 Fixes #46

### CODE REVIEW

**Comment by ThomasKerman** [comment]
LGTM

merging

File: emails/password-reset [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Hello from Kerbal Stuff, {{user.username}}! Someone, probably you, has asked us to reset the password on your account. If that was indeed you, click this link to proceed:
+Hello from {{ site_name }}, {{user.username}}! Someone, probably you, has asked us to reset the password on your account. If that was indeed you, click this link to proceed:
```
Comment (ILM126) [inline emails/password-reset]: I think this is supposed to be "site-name".
That may have been the reason why some bits weren't working.

File: emails/mod-updated [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Hi there! {{ username }} has just published version {{ latest.friendly_version }} of {{ mod.name }} on KerbalStuff! {{#changelog}}Here are the changes:
+Hi there! {{ username }} has just published version {{ latest.friendly_version }} of {{ mod.name }} on {{ site_name }}! {{#changelog}}Here are the changes:
```
Comment (ILM126) [inline emails/mod-updated]: "site-name"

File: emails/grant-notice [hunk 3]
```diff
@@ -1,4 +1,4 @@
-Hello, {{user.username}}. {{mod.user.username}} has asked you to co-author {{mod.name}} on Kerbal Stuff.
+Hello, {{user.username}}. {{mod.user.username}} has asked you to co-author {{mod.name}} on {{ site_name }}.
```
Comment (ILM126) [inline emails/grant-notice]: "site-name"

Comment (ThomasKerman) [inline emails/grant-notice]: No, because 'site-name' is the value in the config, we reassign them to 'site_name' internally when we pass the vars to flask

**Comment by ILM126** [comment]
@ThomasKerman Alright then, wasn't aware of this. Sorry ^^"

**Pull request merged**